packages/service-utils/src/core/usageV2.ts

```diff
@@ -8,10 +10,6 @@ export interface UsageV2Event {
    * The event timestamp. Defaults to now().
    */
   created_at?: Date;
-  /**
-   * The source of the event. Example: "storage"
-   */
-  source: string;
```

Comment:
This is no longer needed because events are sent to product-specific topics (and stored to product-specific tables).